backend/driver.py

```diff
@@ -282,6 +283,30 @@ def send_columns():
         print(traceback.format_exc())
         return send_traceback_error()
 
+@app.route("/api/executiontable", methods=["POST"])
```

Comment:
unsure how the name of this route maps to its functionality. 

2 ways to resolve:

1. documentation abt function

2. choose a better name for the backend route